ansible/roles/nessus/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+# tasks file for nessus
+
+- name: Stop the Nessus service
+  service:
+    name: nessusd
+    state: stopped
+
+#
+# Check if a license key is registered and if not register one
+#
+- name: Check if a license key is registered
+  command: /opt/nessus/sbin/nessuscli fetch --check
+  register: registered
+  failed_when: registered.rc not in [0, 1]
+
+# - name: Register a license key if necessary
+#   command: /opt/nessus/sbin/nessuscli fetch --register-only <activationcode>
```

Comment:
It is unclear to me how we want to store license keys and distribute them.

Reply:
Since license keys are one and done, they're not too sensitive.  I'd think a sidecar yml file would be fine.  I used to associate them to the hosts in the old ansible inventory file.  We could define a variable in terraform, not set a default, and it will prompt the user to input it.

Reply:
Not a bad idea.  We could define said variable in the `production.tfvars` file, so the user isn't prompted in that case.

Reply:
This change has been made.